src/main/scala/com/advancedtelematic/director/data/Codecs.scala

```diff
@@ -41,6 +41,9 @@ object Codecs {
   implicit val decoderCustomImage: Decoder[CustomImage] = deriveDecoder
   implicit val encoderCustomImage: Encoder[CustomImage] = deriveEncoder
 
+  implicit val multiTargetUpdateRowDecoder: Decoder[MultiTargetUpdateRow] = deriveDecoder
```

Comment:
MultiTargetUpdateRow is a db data type and should not ever leak to the client, hence the Row at the end of the type.

I think we should create a new datatype for this if one does not exist yet, take a look at `MultiTargetUpdateRequest`, that is what the client uses to give us MTUs, we should probably deliver something similar. IIRC we already use `LaunchedMultiTargetUpdate` to return launched MTUs, maybe we can reuse that

Reply:
Changed return type to `MultiTargetUpdateRequest ` so we return the same format that we get.